README.md

```diff
@@ -166,6 +166,10 @@ Each mapping should be setup in the following structure:
 - [CWE](mappings/cwe/cwe.json)
 - [Remediation Advice](mappings/remediation_advice/remediation_advice.json)
 
+#### Remediation Training
+- [Source Code Warriors](https://github.com/bugcrowd/vulnerability-rating-taxonomy/commit/ff2999c74ac4f10a6346edd2becfc78a730a09ae/checks?check_suite_id=361261433)
```

Comment:
This is referencing the artifact via the commit hash. Will this link need to be updated for every build ?

Reply:
for now yes, but only on a new release.  before we get to that I want to build a little meta page that gets generated and committed with the artifact, so the link will be removed from this page.

Reply:
updated this to just point to the main tab temporarily and mention that it is a temporary holding place before we make the meta page.